### PR TITLE
Flipkart clone created with responsive design

### PR DESCRIPTION
**Pull request by MS-0712**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,557 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- <meta http-equiv="refresh" content=""> -->
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
```
Comment (rutvik-simformsolutions) [inline index.html]: Instead of providing cdn here, we can practice placing it locally.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+</head>
+ti
+<body>
+    <nav class="navbar navbar-expand-lg navbar-dark px-lg-5" style="background-color: #2874f0;">
+        <div class="container-fluid d-flex justify-content-between px-lg-5 px-sm-0">
+            <div class="col-lg-6 d-flex justify-content-between">
+                <div class="col-auto text-end" style="margin-right: 30px;">
+                    <img src="https://static-assets-web.flixcart.com/fk-p-linchpin-web/fk-cp-zion/img/flipkart-plus_8d85f4.png"
+                        alt="" style="height: 20px;">
+                    <br>
```
Comment (rutvik-simformsolutions) [inline index.html]: Adding "br" tag is not a good practice.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+                    <a class="text-light mt-0 text-decoration-none align-middle" href="#"
+                        style="font-size: smaller; font-style: italic;">Explore Plus <img
+                            src="https://static-assets-web.flixcart.com/fk-p-linchpin-web/fk-cp-zion/img/plus_aef861.png"
+                            class="m-0 pb-1" style="height: 16px;" alt=""></a>
+                </div>
+                <!-- <form class="d-flex justify-content-between"> -->
```
Comment (rutvik-simformsolutions) [inline index.html]: Remove unwanted commented code.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,557 @@
...
+                                More
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li><a class="dropdown-item" href="#">Nottifications</a></li>
+                                <li><a class="dropdown-item" href="#">Customer Care</a></li>
+                            </ul>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="#"><i class="bi bi-cart-fill"
+                                    style="margin-right: 8px;"></i>Cart</a>
```
Comment (rutvik-simformsolutions) [inline index.html]: Adding inline css is not a good idea and is highly unacceptable. Please make another css file and place this css there.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,557 @@
...
+            </div>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+        </div>
+    </nav>
+    <div class="container-fluid px-0 mt-0">
+        <div class="row mt-0 mx-0 px-5" style="background-color: #fff;" id="icons">
```
Comment (rutvik-simformsolutions) [inline index.html]: Remove inline css

File: index.html [hunk 6]
```diff
@@ -0,0 +1,557 @@
...
+                </button>
+                <button class="carousel-control-next" type="button" data-bs-target="#products" data-bs-slide="next">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="visually-hidden">Next</span>
+                </button>
+            </div>
+        </div>
+        <br>
+        <div class="row mx-lg-5" style="background-color: white;">
+            <h4 class="mt-3" style="margin-left: 18px;">Beauty, Food, Toys and more</h4>
```
Comment (rutvik-simformsolutions) [inline index.html]: Remove inline css.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,557 @@
...
+                    <span class="visually-hidden">Previous</span>
+                </button>
+                <button class="carousel-control-next" type="button" data-bs-target="#products" data-bs-slide="next">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="visually-hidden">Next</span>
+                </button>
+            </div>
+        </div>
+        <br>
+        <div class="row mx-lg-5" style="background-color: white;">
```
Comment (rutvik-simformsolutions) [inline index.html]: Remove inline css.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,557 @@
...
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.6.3.min.js"
+        integrity="sha256-pvPw+upLPUjgMXY0G+8O0xUf+/Im1MZjXxxgOcBQBXU=" crossorigin="anonymous"></script>
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous">
+    </script>
+
+    <script>
```
Comment (rutvik-simformsolutions) [inline index.html]: Place this script in a separate file and add link here.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,557 @@
...
+                <button class="carousel-control-next" type="button" data-bs-target="#beauty" data-bs-slide="next">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="visually-hidden">Next</span>
+                </button>
+            </div>
+        </div>
+        <br>
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.6.3.min.js"
```
Comment (rutvik-simformsolutions) [inline index.html]: Please place it locally

File: index.html [hunk 10]
```diff
@@ -0,0 +1,557 @@
...
+        integrity="sha256-pvPw+upLPUjgMXY0G+8O0xUf+/Im1MZjXxxgOcBQBXU=" crossorigin="anonymous"></script>
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
```
Comment (rutvik-simformsolutions) [inline index.html]: Please place it locally.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,557 @@
...
+                        },
+                        500
+                    );
+                }
+            });
+            $("#products .carousel-control-prev").on("click", function () {
+                // console.log('prev from product');
+                // console.log(carouselWidth)
+                // console.log(cardWidth)
+                // console.log(scrollPosition)
```
Comment (rutvik-simformsolutions) [inline index.html]: Remove unwanted commented code.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,557 @@
...
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous">
+    </script>
+
+    <script>
+        var multipleCardCarousel = document.querySelector("#products");
+        var scrollPosition = 0;
+        if (window.matchMedia("(min-width: 768px)").matches) {
+            var carousel = new bootstrap.Carousel(multipleCardCarousel, {
```
Comment (rutvik-simformsolutions) [inline index.html]: Why you have created var here? At most of the place you have created var variable.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,557 @@
...
+            console.log(distance);
+           
+            // Time calculations for days, hours, minutes and seconds
+            var days = Math.floor(distance / (1000 * 60 * 60 * 24));
+            var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+            var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+            var seconds = Math.floor((distance % (1000 * 60)) / 1000);
+
+            // Output the result in an element with id="demo"
+            document.getElementById("timer").innerHTML = days + "d " + hours + "h " +
```
Comment (rutvik-simformsolutions) [inline index.html]: Why you have written javascript code? You have already used jquery then why you have used javascript here. Provide some explanation here.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,557 @@
...
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="icon" href="./images/logo.png" type="image/icon type">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="./css/main.css">
+    <title>Online Shopping Site for Mobiles, Electronics, Furniture, Grocery, Lifestyle, Books & More. Best Offers!
+    </title>
+</head>
+ti
```
Comment (rutvik-simformsolutions) [inline index.html]: Remove Unwanted Code

File: index.html [hunk 15]
```diff
@@ -0,0 +1,557 @@
...
+
+        // Update the count down every 1 second
+        var x = setInterval(function () {
+
+            // Get today's date and time
+            var now = new Date().getTime();
+
+            // Find the distance between now and the count down date
+            var distance = countDownDate - now;
+            console.log(distance);
```
Comment (rutvik-simformsolutions) [inline index.html]: remove console log

File: index.html (hunk 13 above)
Comment (MS-7121) [reply index.html]: Actually I have taken a reference for the timer function from external sources. I found implementation with javascript first and I added that in my code. No other specific reasons. But if you say so, I'll change it to the JQuery.

**Review by rutvik-simformsolutions — APPROVED** [review]

**Pull request merged**